editor/plugins/control_editor_plugin.cpp

```diff
@@ -160,6 +164,42 @@ void EditorPropertyAnchorsPreset::_set_read_only(bool p_read_only) {
 	options->set_disabled(p_read_only);
 }
 
+void EditorPropertyAnchorsPreset::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE:
+		case NOTIFICATION_THEME_CHANGED: {
+			for (int i = 0; i < options->get_item_count(); i++) {
+				if (options->is_item_separator(i)) {
+					continue;
+				}
+				int64_t preset = options->get_item_metadata(i);
+				if (preset < 0 || PRESET_FULL_RECT < preset) {
+					continue;
+				}
+				static const StringName icon_names[PRESET_FULL_RECT + 1] = {
```

Comment:
The size doesn't need to be specified if you are initializing.
```suggestion
				static const StringName icon_names[] = {
```